portal/config/model_persistence.py

```diff
@@ -74,8 +75,9 @@ def persistence_filename(self):
 
     @staticmethod
     def _log(msg):
-        current_app.logger.info(msg)
-        trace(msg)
+        msg = msg.encode('utf-8')
```

Comment:
I don't think you should need this if you're importing `unicode_literals`, what specific log entry/string is it failing on?

Reply:
The error I'm getting is: `UnicodeEncodeError: 'charmap' codec can't encode character '\u2713' in position 619: character maps to <undefined>`

And the string it's failing on is: `Message: "User 1 collision on import.  added {'id': 1}\nreplace {'status': 'registered'} with {'status': 'unknown'}\nreplace {'identifier': [{'value': '1', 'system': 'http://us.truenth.org/identity-codes/TrueNTH-identity', 'use': 'official'}, {'value': 'testy@ex
ample.com', 'system': 'http://us.truenth.org/identity-codes/TrueNTH-username', 'use': 'secondary'}]} with {'identifier': [{'value': 'None', 'system': 'http://us.truenth.org/identity-codes/TrueNTH-identity', 'use': 'official'}]}\nreplace {'photo': [{'url': 'http:
//examle.com/photo.jpg'}]} with {'photo': []}\nreplace {'name': {'given': '\u2713', 'family': 'Last'}} with {'name': {'given': None, 'family': None}}\n"`